denizen_scripts/hub1/repo-link/Adriftus_Moderator_Panel/panels/mod_panels_events.dsc

```diff
@@ -0,0 +1,20 @@
+# -- GLOBAL INVENTORY EVENTS --
+mod_global_inv_events:
+  type: world
+  debug: false
+  events:
+    on player clicks item in mod_*_inv priority:10:
+      - determine cancelled
+    
+    on player drags item in mod_*_inv priority:10:
+      - determine cancelled
+
+    on player clicks red_stained_glass_pane in mod_*_inv:
+      - if <context.item.has_nbt[to]>:
+        - choose <context.item.nbt[to]>:
+          - case actions:
+            - inventory open d:<inventory[mod_actions_inv]>
```

Comment:
When using script or noted names, you don't need a wrapper. this can just be `inventory open d:mod_actions_inv`